src/common/types/payloads/seo-payload.type.ts

```diff
@@ -1,4 +1,4 @@
-import { SeoProps } from '../seo/seo-props';
+import { SeoProps } from '../props/seo-props.type';
```

Comment:
remove props folder, seo props to seo folder, component props to something like layout